okta.json

```
@@ -54,7 +55,7 @@
         {
             "action": "send push notification",
             "description": "Validate the user prompt with push notification",
-            "verbose": "The action has not yet been implemented for the \"sms\" and the \"token:software:totp\" factortypes.<br>Currently, the action would generate a new OTP for the \"sms\" factortype and will not verify it.<br>The action would fail for the \"token:software:totp\" factortype as it has not yet been implemented.",
+            "verbose": "The action has not yet been implemented for the \"sms\" and the \"token:software:totp\" factortypes.<br>Currently, the action would generate a new OTP for the \"sms\" factortype and will not verify it.<br>The action would fail for the \"token:software:totp\" factortype as it has not yet been implemented.<br>Either of the below roles are need to run this action.<ui><li>SUPER_ADMIN</li><li>ORG_ADMIN</li><li>USER_ADMIN</li><li>HELP_DESK_ADMIN</li></ui>",
```

```suggestion
            "verbose": "The action has not yet been implemented for the \"sms\" and the \"token:software:totp\" factortypes.<br>Currently, the action would generate a new OTP for the \"sms\" factortype and will not verify it.<br>The action would fail for the \"token:software:totp\" factortype as it has not yet been implemented.<br>One of the following roles is required to run this action:<ui><li>SUPER_ADMIN</li><li>ORG_ADMIN</li><li>USER_ADMIN</li><li>HELP_DESK_ADMIN</li></ui>",
```

same change for all the other places where this same text is used

---

Change as per comment.